lib/CampMinder/base64.rb

```diff
@@ -0,0 +1,15 @@
+require 'base64'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.